src/library/components/resource-type.js

```diff
@@ -8,7 +8,8 @@ const ResourceType = Component({
     const materialTypeLabels = {
       'Interactive': 'model',
       'Activity': 'activity',
-      'Investigation': 'sequence'
+      'Investigation': 'sequence',
+      'Project': 'collection'
```

Comment:
Perhaps this has to change since the Portal code has changed to not use Project.